<launch>
  <arg name="use_machine" default="true" />
  <arg name="machine" default="c1" />
  <arg name="db_path" default="/var/lib/robot/mongodb_store" />
  
  <include file="$(find mongodb_store)/launch/mongodb_store.launch">
    <arg name="use_machine" value="$(arg use_machine)" />
    <arg name="machine" value="$(arg machine)"/>
    <arg name="db_path" value="$(arg db_path)" />
    <arg name="port" value="62345" />
    <arg name="use_repl_set" value="true" />
    <arg name="repl_set" value="rs0" />
  </include>

  <node name="mongod_watcher" pkg="jsk_pr2_startup" type="mongod_watcher.py">
    <param name="db_path" value="$(arg db_path)" />
  </node>

  <node name="move_base_db" pkg="jsk_pr2_startup" type="move_base_db.py">
    <param name="map_frame" value="eng2"/>
    <param name="robot_frame" value="base_footprint"/>
    <param name="update_cycle" value="0.1" />
  </node>
  <!--node name="action_result_db" pkg="jsk_pr2_startup" type="action_result_db.py"/-->
  <group ns="/openni_c2/rgb">
     <node name="object_detection_db" pkg="jsk_pr2_startup" type="objectdetection_db.py" if="false"/>
  </group>
  <include file="$(find jsk_pr2_startup)/jsk_pr2_lifelog/pr2_tweet.launch" if="false">
    <arg name="machine" value="c1" />
  </include>
</launch>
